lib/modules/side_menu/side_menu_view.dart

```diff
@@ -0,0 +1,43 @@
+part of 'side_menu_module.dart';
+
+abstract class SideMenuViewDelegate {
+  BehaviorSubject<void> get logoutButtonDidTap;
```

Comment:
as we had a long discussion before, don't you plan to change this way of exposing the state by a function?

`void logoutButtonDidTap()` instead of `logoutButtonDidTap.add(null)` ? Since it's a new file I think you can avoid doing the same mistake as before.

Reply:
@sleepylee in fact, I did all tasks and just waiting for review, so my plan that will do these refactor works at the end 😁